source/dynamical_systems/README.md

```diff
@@ -1,2 +1,209 @@
 # dynamical_systems
 
+This library provides a set of classes to represent **Dynamical Systems**: 
+functions which map a state to a state derivative.
+Those are a set of helpers functions to handle common concepts in robotics such as transformations between frames and 
+the link between them and the robot state.
+This description covers most of the functionalities starting from the spatial transformations.
+
+## Table of contents:
+* [Base DynamicalSystem](#base-dynamicalsystem)
+  * [Base frame](#base-frame)
+* [Linear](#linear)
+  * [Evaluating the Linear DS](#evaluating-the-linear-ds)
+  * [Configuring the Linear DS](#configuring-the-linear-ds)
+  * [Reference frames](#reference-frames)
+* [Circular](#circular)
+
+## Base DynamicalSystem
+
+The base class DynamicalSystem defines the main public interface pattern which the derived classes follow.
+ 
+It is templated for any state type, though the intention is to use the **State Representation** library
+(for example, `CartesianState` or `JointState`).
+
+The main function is:
+```c++
+template<class S>
+S DynamicalSystem<S>::evaluate(const S& state) const
+```
+where `S` is the state representation type.
+The purpose of the evaluate function is to calculate a state derivative from a state.
+The current implementations support calculating a velocity from a position in either joint space or Cartesian space,
+though higher derivatives and other space types can be extended in a similar fashion.
+
+In the case of a `CartesianState`, only the position and orientation is evaluated, and the returned object
```

Comment:
I think if you mention CartesianState specifically, you should also say what happens in the case of a JointState